lib/membrane/rtp/session_bin.ex

```diff
@@ -649,6 +654,46 @@ defmodule Membrane.RTP.SessionBin do
     {{:ok, forward: {:twcc_sender, msg}}, state}
   end
 
+  @impl true
+  def handle_other(%RequireExtensions{} = msg, _ctx, state) do
+    {{:ok, forward: {:ssrc_router, msg}}, state}
+  end
+
+  @impl true
+  def handle_other(%RTXInfo{ssrc: ssrc} = msg, ctx, state) do
+    rtx_parser_opts = %RTP.RTXParser{
+      rtx_payload_type: msg.rtx_payload_type,
+      payload_type: msg.original_payload_type,
+      rid_id: msg.rid_id,
+      repaired_rid_id: msg.repaired_rid_id
+    }
+
+    links_generator = fn twcc? ->
+      [
+        link(:ssrc_router)
+        |> via_out(Pad.ref(:output, ssrc))
+        # TODO: Fix TWCCReceiver not noticing packets dropped by SSRCRouter
+        |> then(&link_twcc_receiver_if(twcc?, &1, ssrc))
+        |> to(
+          {:decryptor, ssrc},
+          struct(Membrane.SRTP.Decryptor, %{policies: state.srtp_policies})
```

Comment:
I forgot about this in the first round of CR but  the  `SRTP.Decryptor` module is optional. While I believe there is no point in using RTX without encryption, maybe we should check against `secure?` option being set  and code of `ExLibSRTP` being loaded?